easybuild/easyconfigs/q/Qhull/Qhull-2012.1-intel-2014b.eb

```diff
@@ -15,12 +15,20 @@ toolchain = {'name': 'intel', 'version': '2014b'}
 sources = ['%(namelower)s-%(version)s-src.tgz']
 source_urls = ['http://www.qhull.org/download/']
 
-patches = ['Qhull-2012.1-intel-fix.patch']
+patches = [
+    'Qhull-2012.1-intel-fix.patch',
+    ('Qhull_pkgconfig.patch', 1),
+]
 
-builddependencies = [('CMake', '2.8.12')]
+builddependencies = [('CMake', '3.2.1', '', ('GNU', '4.9.3-2.25'))]
+
+postinstallcmds = [
+    "cat qhull.pc.in | sed 's@INSTALLDIR@%(installdir)s@g' | sed 's@VERSION@%(version)s@g' > %(builddir)s/qhull.pc",
```

Comment:
The `cat` can go anyway since you can give the filename as an arg to the first `sed`.